taxonomies/ingredients.txt

```diff
@@ -24897,6 +24931,16 @@ pt:kamut
 ru:камут
 sl:kamut
 sv:kamut
+wikidata:en:Q11163068
+wikipedia:en:https://en.wikipedia.org/wiki/Khorasan_wheat
+
+<en:khorasan wheat
```

Comment:
is `khorosan wheat` not a child of `cereal wheat`? Or should be? Then it can be left out.

Reply:
the existing entry for kamut (which i added the missing english translation for) was just under "cereal", and I didn't change that.